### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  grid-column: span 1;
+  grid-row: span 3;
+}
+
+@media (max-width: 990px) {
+  .container {
+    grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+    grid-template-rows: minmax(auto, auto);
+  }
+  .container .box {
```
Comment (anask601) [inline style.css:86]: @nehalskdev Can you please explain why we are adding unset property here

**Review by murtazshaikh — APPROVED** [review]
LGTM!

File: style.css (hunk 1 above)
Comment (nehalskdev) [reply style.css:86]: @anask601 we have added ```unset``` property here so the column and row inherit the default behavior.

Comment (nehalskdev) [reply style.css:86]: ```grid-template-rows``` is used to explicitly define the structure of the grid, while ```grid-auto-rows``` is used to set the default size of rows that are not explicitly defined.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap");
```
Comment (Abbas4tech) [inline style.css:1]: What's the other way of installing 3rd party fonts ??

File: style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  min-height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-self: center;
+
+  background: #222;
+}
+.container {
+  margin: 40px;
```
Comment (Abbas4tech) [inline style.css]: Please use relative units

File: style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  text-align: center;
+  transition: 0.5s;
+}
+.container .box:hover {
+  background: #e91e63;
+}
+.container .box img {
+  position: relative;
+  max-width: 100px;
+  margin: 0 0 10px 0;
```
Comment (Abbas4tech) [inline style.css]:
```suggestion
  margin-bottom:10px;
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+@media (max-width: 990px) {
+  .container {
+    grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+    grid-template-rows: minmax(auto, auto);
+  }
+  .container .box {
+    grid-row: unset !important;
+    grid-column: unset !important;
```
Comment (Abbas4tech) [inline style.css:88]: Why do we need `!important` declaration here ??

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Need some refactoring

File: style.css (hunk 2 above)
Comment (nehalskdev) [reply style.css:1]: ```@font-face``` Rule as far as i know i am studying about other methods too

File: style.css (hunk 3 above)
Comment (nehalskdev) [reply style.css]: added thanks @Abbas4tech

File: style.css (hunk 5 above)
Comment (nehalskdev) [reply style.css:88]: because wanted to make the ```unset``` value applied no matter what

**Pull request merged**